packages/common/src/index.ts

```diff
@@ -59,6 +59,7 @@ export enum Hardfork {
   Berlin = 'berlin',
   London = 'london',
   Shanghai = 'shanghai',
+  TheMerge = 'theMerge',
```

Comment:
@holgerd77 what do you think about just calling it `merge` for short?

Reply:
either `merge` or `theMerge(tm)`  Them `(tm)` is important!

Reply:
Ok, let's do `merge` 😛 😅 (feel free everyone to update on occasion).